source/deployment/video-and-audio-calling.rst

```diff
@@ -11,62 +11,78 @@ Zoom
 ~~~~~~~~~~~~~~~~~~~~
 
 - Start and join voice calls, video calls and use screen sharing with your team members, developed by Mattermost.
-- Clicking a video icon in a Mattermost channel invites team members to join a Zoom call, hosted using the credentials of the user who initiated the call.
+- Click a video icon in a Mattermost channel to invite team members to join a Zoom call, hosted using the credentials of the user who initiated the call.
 - Requires 1 paid `Zoom Pro, Business, Education, or API plan, <https://zoom.us/pricing>`_ to generate an API key and secret for $14.99/month.
 - Supports a self-hosted cloud solution, and a vendor-hosted cloud solution.
-- Docs: https://docs.mattermost.com/integrations/zoom.html
-- Source code: https://github.com/mattermost/mattermost-plugin-zoom
+- Docs available at: https://docs.mattermost.com/integrations/zoom.html
+- Source code available at: https://github.com/mattermost/mattermost-plugin-zoom
+
+Cisco Webex Cloud
+~~~~~~~~~~~~~~~~~~~~
+
+- Start and join voice and video calls, and share your screen with members of your team, developed by Mattermost.
```

Comment:
I kept `developed by` without brackets, did you feel strongly about this?

Reply:
No, I'm a 1/5 so that works for me. :)